wercker-step.yml

```diff
@@ -1,3 +1,3 @@
 name: rails-database-yml
-version: 1.0.2
```

Comment:
@ae06710 Because we intend to publish this repos as our step. It's ok to sync version with wercker repos, I think.  But when we publish a wecker step, we can't use same version twice. Then there is a possible that version is not match between ours and theirs. Anyway I agree with what you say and I'll modify it.